packages/valory/customs/asset_lending/asset_lending.py

```diff
@@ -325,48 +326,82 @@ def calculate_il_risk_score_for_silos(token0_symbol, silos, coingecko_api_key):
 def calculate_metrics(current_pool: Dict[str, Any], coingecko_api_key: str, **kwargs) -> Optional[Dict[str, Any]]:
     il_risk_score = calculate_il_risk_score_for_silos(current_pool.get("token0_symbol"), current_pool.get('whitelistedSilos',[]), coingecko_api_key)
     sharpe_ratio = get_sharpe_ratio_for_address(current_pool['pool_address'])
-
+    liquidity_risk_metric = analyze_vault_liquidity(current_pool['pool_address'])
     return {
         "il_risk_score": il_risk_score,
-        "sharpe_ratio": sharpe_ratio
+        "sharpe_ratio": sharpe_ratio,
+        "liquidity_risk_metric":liquidity_risk_metric
```

Comment:
add two separate metrics, depth score and max_position_size instead of liquidity_risk_metric